app/src/pages/instance/organizationsPage/organizationsPageHeader/organizationsPageHeader.jsx

```diff
@@ -65,6 +71,11 @@ export const OrganizationsPageHeader = ({
                 event={ORGANIZATION_PAGE_EVENTS.SEARCH_ORGANIZATION_FIELD}
               />
               <i className={cx('filters-icon')}>{Parser(filterIcon)}</i>
```

Comment:
Right now we have two buttons.
Looks like one should be removed 
![image](https://github.com/user-attachments/assets/173fc23d-66ee-42ed-90d5-81be5fa96859)

Reply:
Done.